charset_normalizer/version.py

```diff
@@ -2,5 +2,5 @@
 Expose version
 """
 
-__version__ = "2.0.12"
+from ._version import __version__
```

Comment:
That is going to be problematic as it require to generate a file prior to loading the package.